nservicebus/pipeline/features.md

```diff
@@ -16,21 +16,21 @@ related:
  - samples/startup-shutdown-sequence
 ---
 
-While NServiceBus provides some simple interfaces to plug in the code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
+While NServiceBus provides interfaces to plug in code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
 
 Features allow:
 
- * Configure required dependencies for the Feature.
- * Enable or disable a Feature via configuration or based on conditions and dependencies.
- * Get full access to the pipeline, settings and the container.
+ * Configure required dependencies.
+ * Enabling or disabling via configuration or based on conditions and dependencies.
+ * Full access to the pipeline, settings and the container.
```

Comment:
You're not consistent in the list, choose one option and stick to it :)
e.g. configuring, enabling, accessing, etc.